src/App.js

```diff
@@ -5,26 +5,27 @@ import './App.css';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import Postlist from './PostList/PostList';
```

Comment:
`Postlist` -> `PostList`
if a class name consists of two or more words, you need to separate them